src/components/elements/Text.jsx

```diff
@@ -1,6 +1,18 @@
+// Available size options(String):
+//   - 'small'
+//   - 'normal'
+//   - 'large'
+// Available color options(String):
+//   - 'cyan'
+//   - 'black'
+// Available sort options(String):
+//   - 'none'
+//   - 'longUrl'
 import style from './Text.module.css';
 
-function Text({ children }) {
-  return <p className={style.text}>{children}</p>;
+function Text({ children, color = 'black', size = 'normal', sort = 'none' }) {
+  return (
+    <p className={`${style.text} ${style[color]} ${style[size]} ${style[sort]}`}>{children}</p>
```

Comment:
Use `clsx` library to handle this amount of styles. Here and in other files, where you have more than one className 

https://www.npmjs.com/package/clsx